aten/src/ATen/native_parse.py

```diff
@@ -277,8 +264,11 @@ def is_tensor_option(argument):
             if len(topt_representation) == number_of_arguments:
                 merged_argument = check_topt_representation(topt_representation)
                 assert merged_argument, \
-                    "Unsupported combination of TensorOptions {}, the only currently supported combinations are {}"\
-                    .format(str(topt_representation), str(supported_topt_arguments))
+                    "Unsupported combination of TensorOptions in {}:\n{}\n\n"\
+                    "The only currently supported combinations are:\n{}"\
```

Comment:
I think this should now just be "the only currently supported combination is", right? 

...that is if we don't want to make it even sterner, like "TensorOptions must appear in exactly the sequence {}" or something.

I'd call it a nit, but it might be confusing to someone who's new and doesn't really know how to interpret a list of half dozen individual arguments along with an inadvertent plural in the error message.

Reply:
I guess I'll let you handle this one, since this PR also needs to get the BC test script updated too.